File: frontend/src/app/model/sortDTOCottage.ts
import { InhrCottageDTO } from "./inhrCottageDTO";

export class SortDTOCottage {
    private dto: InhrCottageDTO[];
    private sortParam: string;

	constructor($dto: InhrCottageDTO[], $sortParam: string) {
		this.dto = $dto;
		this.sortParam = $sortParam;
	}

}

File: isa/src/main/java/com/isa/services/dto/SortDTOCottage.java
package com.isa.services.dto;

import java.util.List;

public class SortDTOCottage {
    List<InhrCottageDTO> dto;
    String sortParam;

    public SortDTOCottage(){}

    public SortDTOCottage(List<InhrCottageDTO> dto, String sortParam) {
        this.dto = dto;
        this.sortParam = sortParam;
    }

    public List<InhrCottageDTO> getDto() {
        return dto;
    }

    public void setDto(List<InhrCottageDTO> dto) {
        this.dto = dto;
    }

    public String getSortParam() {
        return sortParam;
    }

    public void setSortParam(String sortParam) {
        this.sortParam = sortParam;
    }
}
